release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -38,7 +38,14 @@ Due to timing (see below)
 
 Weeks: -2 to 0
 
-At least 2 weeks before the new release cycle begins, the EA is responsible for posting the "Assemble the X.Y Release Team" issue in the SIG-Release repo ([example](https://github.com/kubernetes/sig-release/issues/776)).  They should then make sure that the new Release Lead is officially nominated and chosen, and then issue a call for nominations/volunteers for the other Role Leads.
+At least 2 weeks before the new release cycle begins, the EA is responsible for
+posting the "Assemble the X.Y Release Team" issue in the SIG-Release repo
```

Comment:
```suggestion
posting the "Assemble the X.Y Release Team" issue in the SIG Release repo
```

I know this was in the original text, but calling out that we should not hyphenate SIG names, outside of cases that might require non-whitespace characters e.g., the explicit name of the GitHub repo (`github.com/kubernetes/sig-release`)